service/handlers.go

```diff
@@ -15,6 +15,18 @@ import (
 
 var pubkey *rsa.PublicKey
 
+// JWT Auth related constants
+const (
+	// context
+	ContextToken = "token"
+	ContextClaims = "claims"
+
+	// claims
+	ClaimUsername = "username"
+	ClaimUserId = "userid"
```

Comment:
Just a general thing about Go naming conventions: `ClaimUserID` instead of `ClaimUserId`.